OpenRA Debian package creates ~/.openra owned by root

On a machine with no previous installation of OpenRA, I installed the `openra_playtest.20170930_all.deb` from the Downloads page.

On first running `openra-ra` I received the stack trace:
```
[ERROR] FATAL UNHANDLED EXCEPTION: System.UnauthorizedAccessException: Access to the path "/home/james/.openra/Logs" is denied.
  at System.IO.Directory.CreateDirectoriesInternal (System.String path) [0x00070] in <8f2c484307284b51944a1a13a14c0266>:0 
  at System.IO.Directory.CreateDirectory (System.String path) [0x000a4] in <8f2c484307284b51944a1a13a14c0266>:0 
  at OpenRA.Log+<FilenamesForChannel>c__Iterator0.MoveNext () [0x00037] in <e9a530e678184aae8f56cb54e517f6ce>:0 
  at OpenRA.Log.AddChannel (System.String channelName, System.String baseFilename) [0x000b1] in <e9a530e678184aae8f56cb54e517f6ce>:0 
  at OpenRA.Program.FatalError (System.Exception ex) [0x00028] in <e9a530e678184aae8f56cb54e517f6ce>:0 
  at OpenRA.Program.Main (System.String[] args) [0x0005d] in <e9a530e678184aae8f56cb54e517f6ce>:0
```

This is because the `~/.openra` directory is being created with owner `root`:

```
$ ls -la .openra/
total 8
drwxr-xr-x  2 root  root  4096 Oct  6 16:55 .
drwxr-xr-x 59 james james 4096 Oct  6 17:15 ..
```

`chown`ing the directory to the correct user allows OpenRA to run.

I'm assuming that it's an issue with the `deb` package rather than the code itself since `~/.openra` is created on package installation rather than the first running of `openra-*`.

--- a/OpenRA.Game/ExternalMods.cs
+++ b/OpenRA.Game/ExternalMods.cs
@@ -123,9 +123,6 @@ namespace OpenRA
 
 			foreach (var source in sources.Distinct())
 			{
-				if (!Directory.Exists(source))
-					continue;
-
 				var metadataPath = Path.Combine(source, "ModMetadata");
 
 				try

--- a/OpenRA.Game/Platform.cs
+++ b/OpenRA.Game/Platform.cs
@@ -93,9 +93,6 @@ namespace OpenRA
 					break;
 			}
 
-			if (!Directory.Exists(dir))
-				Directory.CreateDirectory(dir);
-
 			return dir + Path.DirectorySeparatorChar;
 		}
 

